client/src/Pages/Instruct-Page/Instruct-Page.css

```diff
@@ -2,6 +2,7 @@
 h1,
 p {
     text-align: center;
+    color: #66fcf1;
```

Comment:
We should really make these colors into custom properties. 

https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties